src/dali_backend.cc

```diff
@@ -436,14 +444,30 @@ TRITONBACKEND_ModelInstanceExecute(
           "Unknown DALI Backend error");
     }
 
+    exec_end_ns = detail::capture_time();
+
+    RETURN_IF_ERROR(TRITONBACKEND_ModelInstanceReportStatistics(
+        instance, requests[i], !error, exec_start_ns,
+        request_meta.compute_start_ns, request_meta.compute_end_ns,
+        exec_end_ns));
+
     RETURN_IF_ERROR(TRITONBACKEND_ResponseSend(
         responses[i],
         TRITONSERVER_ResponseCompleteFlag::TRITONSERVER_RESPONSE_COMPLETE_FINAL,
         error));
     RETURN_IF_ERROR(TRITONBACKEND_RequestRelease(
         requests[i],
         TRITONSERVER_RequestReleaseFlag::TRITONSERVER_REQUEST_RELEASE_ALL));
+
+    total_batch_size += request_meta.batch_size;
+    batch_exec_end_ns = exec_end_ns;
+    batch_compute_end_ns = request_meta.compute_end_ns;
   }
+
+  RETURN_IF_ERROR(TRITONBACKEND_ModelInstanceReportBatchStatistics(
+      instance, total_batch_size, batch_exec_start_ns, batch_compute_start_ns,
```

Comment:
Yeah, I've missed it.
Fixed setting the `batch_...`. At one point I needed to use the ternary instead of a simple if, cause the compiler was complaining about `batch_compute_start_ns may be used uninitialized in this function`.